beacon-chain/core/state/transition.go

```diff
@@ -255,6 +255,46 @@ func ProcessOperations(
 	ctx, span := trace.StartSpan(ctx, "beacon-chain.ChainService.state.ProcessOperations")
 	defer span.End()
 
+	if uint64(len(body.ProposerSlashings)) > params.BeaconConfig().MaxProposerSlashings {
```

Comment:
Better to refactor these to a separate function? ex: `VerifyOperationLength`?

Reply:
Good idea!